packages/@okta/vuepress-site/guides/sign-into-mobile-app/sign-in-custom-ui/android/signin.md

```diff
@@ -0,0 +1,183 @@
+In this example we will show how to create a custom UI for user sign-in. First we will create a simple dialog to ask for username and password:
```

Comment:
@FeiChen-okta Sorry, meant to sync with you on this: can you remove this from `sign-into-mobile-app` and place it in this guide instead? https://github.com/okta/okta-developer-docs/pull/333